examples/solar_cycle_example.py

```diff
@@ -0,0 +1,40 @@
+"""
+===============
+The Solar Cycle
+===============
+
+This example shows the current and possible next solar cycle.
+"""
+
+import matplotlib.pyplot as plt
+import sunpy.lightcurve as lc
+import datetime
+
```

Comment:
also line breaks between the different sets of imports if we are being *really* picky